server/load.js

```diff
@@ -134,7 +137,15 @@ exports.handleLoad = function(req, res, app, format) {
           filename += '/';
         }
 
-        var list = buildDirList(absfile, fs.readdirSync(absfile).sort());
+        var files = fs.readdirSync(absfile);
```

Comment:
If you do end up using DirCache for all directory listings, we could rename it to something like DirLoader to reflect its role.